doc/install_ubuntu.txt

```diff
@@ -144,13 +164,22 @@ system package, you can run this:
 Bleeding edge
 ~~~~~~~~~~~~~
 
-Do like in the section "Updating Theano", but use
-``git+git://github.com/Theano/Theano.git`` instead of ``theano``.
+Change to the Theano directory and run:
+
+.. code-block: bash
+    
+    git pull
+    python setup.py develop --user
 
 
 Manual Openblas instruction
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. comment::
+
+    I believe this is outdated, my machine seems to be using 8 threads 
+    happily with the binary openblas...
+
```

Comment:
This means that we can probably delete this section, but I haven't personally encountered this so I'll wait a bit to see if somebody else comments, otherwise, do as you wish.